bzzeth/bzzeth.go

```diff
@@ -85,6 +90,15 @@ func (b *BzzEth) handleMsg(p *Peer) func(context.Context, interface{}) error {
 	}
 }
 
+// handleMsgFromSwarmNode is used in the case if this node is connected to a Swarm node
+// If any message is received in this case, the peer needs to be dropped
+func (b *BzzEth) handleMsgFromSwarmNode(p *Peer) func(context.Context, interface{}) error {
+	return func(ctx context.Context, msg interface{}) error {
+		p.logger.Debug("bzzeth.handleMsgFromSwarmNode")
+		return errRcvdMsgFromSwarmNode
```

Comment:
Is it needed to always return this error?

Reply:
yes. no protocol message is legal from another swarm node at this point